lib/deck.rb

```diff
@@ -0,0 +1,24 @@
+class Deck
+
+  attr_reader :cards
+  def initialize(cards)
+     @cards = cards
+
+  end
+
+  def count
+    cards.count
```

Comment:
Be careful when referencing instance variables that you always include the `@` - even though Ruby lets it work without it, it can be confusing in larger, more complex applications.